bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/HistoricalPersistenceStoreService.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.persistence;
+
+/**
+ * This class provides an interface to the a {@link PersistenceService} to allow data to be stored
+ * at a specific time. This allows bindings that interface to devices that store data internally,
+ * and then periodically provide it to the server to be accommodated.
+ *
+ * @author Chris Jackson - Initial implementation and API
+ *
+ */
+public interface HistoricalPersistenceStoreService {
```

Comment:
> But it's not modifyable

Right, but if we combine it with your other "separate interface to come", it will be :-)

> I'm happy to consolidate into a single interface if you prefer, but I recall in the past you wanted to keep admin (delete) functions separate.

Well, I wanted to keep any non-chonological write operations separate. If we introduce a "store with timestamp" method, I think this anyhow means that the service is able to modify its already persisted data - and hence also offers deletion of entries etc. So for me, it makes sense to combine those methods in one interface.